src/tools/illink/src/linker/Linker.Steps/MarkStep.MethodDefinitionDependencyNode.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using ILCompiler.DependencyAnalysisFramework;
+using Mono.Cecil;
+
+namespace Mono.Linker.Steps
+{
+	public partial class MarkStep
+	{
+		internal sealed class MethodDefinitionDependencyNode : DependencyNodeCore<MarkStepNodeFactory>
```

Comment:
Nit: we suffix only `Node` in crossgen/ilc. Less typing.

```suggestion
		internal sealed class MethodDefinitionNode : DependencyNodeCore<MarkStepNodeFactory>
```